src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -238,7 +256,8 @@ public async Task<ResourceWrapper> GetAsync(ResourceKey key, CancellationToken c
                             isDeleted,
                             searchIndices: null,
                             compartmentIndices: null,
-                            lastModifiedClaims: null)
+                            lastModifiedClaims: null,
+                            searchParamHash)
```

Comment:
What is the reason for including this in ResourceWrapper?

Reply:
Ah, good question. From what I understand, it's a logical place to keep it because reindexing operations involve searching for and updating resources, which ingest and output `ResourceWrapper` objects. During a reindexing operation, we need to keep track of the resources that have been indexed as well as their updated hash values, and the resource wrapper is a nice way to bundle things together.

@feordin am I missing anything?

Reply:
@rotodd That is correct. It is another key piece of metadata for the resource, and directly related to the SearchIndices collection which are also a property on the ResourceWrapper.